apps/docs/content/getting-started/animation.mdx

```diff
@@ -0,0 +1,247 @@
+---
+title: Animation
+storybook: '/brand/storybook/?path=/story/components-animations--playground'
+description: Use animations to add visual interest and interactivity to a web page or application.
+---
+
+import {PropTableValues} from '../../src/components'
+import {Link} from 'gatsby'
+import {AnimationVariants, Label} from '@primer/react-brand'
+import {supportedComponents} from '../../scripts/components-with-animation'
+
+Certain Primer Brand components include animation presets. These components are:
+
+<PropTableValues
+  values={supportedComponents}
+  commaSeparated
+  removeApostrophes
+/>
+
+<br />
+<br />
```

Comment:
Maybe wrapping this into <p> to avoid <br>?

```suggestion
<p>
  <PropTableValues
    values={supportedComponents}
    commaSeparated
    removeApostrophes
  />
</p>
```

Reply:
ooo didn't think of that. I'll try it out to make sure it spaces okay.